test/parse/string-utf-escape.txt

```diff
@@ -0,0 +1,2 @@
+;;; TOOL: wat2wasm
+(module (func) (export "a:\u{0}b:\u{00001}c:\u{7f}d:\u{80}e:\u{7ff}f:\u{800}g:\u{ffff}h:\u{10000}i:\u{10ffff}" (func 0)))
```

Comment:
This is a similar file to string-escape.txt, but I couldn't find its corresponding binary expected output file. It would be good to prove the encoding is correct.

Reply:
I am not familiar with the term round tripping. Is that a command?

Reply:
Take a look at `test/roundtrip/`..  I think this test should probably live there.